ntoml/netlify_toml.go

```diff
@@ -0,0 +1,103 @@
+package ntoml
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+	"github.com/pkg/errors"
+)
+
+const DefaultFilename = "netlify.toml"
+
+type NetlifyToml struct {
```

Comment:
This doesn't match the configuration in buildbot. Let's make sure they are compatible:

https://github.com/netlify/buildbot/blob/master/bot/configuration/configuration.go#L19-L31